src/curated-corpus/components/ApprovedItemForm/ApprovedItemForm.tsx

```diff
@@ -32,7 +33,7 @@ interface ApprovedItemFormProps {
   /**
    * The approved item that needs to be edited.
    */
-  approvedItem: ApprovedCuratedCorpusItem;
+  approvedItem: ApprovedCuratedCorpusItem | ApprovedItemFromProspect;
```

Comment:
I've had to create an new type as you'll see below and use it here and also in the `ImageUpload` component. 

This is where we start seeing some effects of coupling since a single component (`ApprovedItemForm`) is being used to accommodate different items/types now. I think it is fine since the other solution of copying this component for a prospect type would be too redundant since the UI would be the exact same.

Reply:
I'm happy with this! All it does is swap the type for the `language` prop for something compatible with the data we get, it's the right approach.